backend/app/api/endpoints/prompt_optimization.py

```
@@ -0,0 +1,37 @@
+):
+    """
+    Apply prompt optimization changes to Ghost or TeamMember.
+
+    Requires Developer+ permission on the target resources.
+    """
+    try:
+        result = apply_prompt_changes(
+            db=db, user=current_user, team_id=request.team_id, changes=request.changes
+        )
+        return result
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)